specification/taf-spec.md

```diff
@@ -0,0 +1,946 @@
+      └─ repo3
+          └─ {"branch": "branch1", "commit": "m2n3o4p5"}
+```
+
+By navigating through the commit history of the authentication repository, it's possible to
+collect all registered commits for each target repository and each of their branches. So, within
+the TAF framework, there's no need to track all past commits of target repositories in a single
+commit of the authentication repository. Similarly, previous versions of metadata files can be
+easily accessed through the Git history.
+
+## [5 Updater](#5-updater)
+
+This section focuses on the updater, a key component of the TAF framework. The updater's main job is
+to securely clone and update both authentication and target repositories. It replaces the need for
+manually running Git commands with its automated process. This is especially important because it
+incorporates a validation step, ensuring that all changes are verified before being applied to the
+user's machine.
+
+As previously discussed, one authentication repository in TAF can reference another. This
+functionality is particularly useful for scenarios where an organization needs to create local
+copies of repositories managed by another organization. Currently, this inter-repository linkage is
+defined using the `dependencies.json` file. Before exploring this use-case in more detail, we will
+first examine the update process for an authentication repository that does not have any such
+dependencies
+
+### [5.1 The update process](#51-the-update-process)
+
+The update process in TAF begins with specific inputs: the file system location of the authentication
+repository, the URL of the remote repository if a local copy does not exist, and optionally, an
+out-of-band commit. The out-of-band commit, which for now is expected to be first commit of the
+authentication repository, is obtained by contacting the official publishers of the repository. If
+this commit is provided, it acts as a verification step to ensure there's no mismatch between the
+expected initial commit and the actual first commit in the remote repository. This verification
+helps to detect scenarios where users might be directed to download a repository from untrustworthy
+sources, such as those created by malicious actors.
+
+The update process in TAF can be broken down into the following steps:
+
+1. Clone the authentication repository as a bare Git repository to a temporary location. Validate its
+compliance with TUF specification at every commit, starting from the previously saved last
+validated commit. Check the out-of-band commit if provided.
+1. Update the user's existing local repository copy or create a new one if it doesn't exist.
+1. Clone target repositories if they are not present locally, or fetch their changes. Do not merge new changes.
+1. Compare the data in the authentication repository with the fetched commits of target repositories.
+1. Execute any scripts designated to run after the update process
```

Comment:
These are still 1.s

Reply:
I didn't see that comment. This is actually correct, if you view the rendered file, it will have the correct enumeration.